install/kubernetes/cilium/values.yaml

```diff
@@ -10,6 +10,7 @@ debug:
 rbac:
   create: true
 
+quickInstall: false
```

Comment:
I don't understand how the user is supposed to interpret or use this flag. Why is it necessary to introduce this flag in order to remove the validation check?

Reply:
If this flag isn't present the new TLS secrets will generate and the quick-install file will receive a randomly generated set of TLS secrets.

Reply:
I could reverse it and get rid of it having to be in the values file.